src/components/SchedulePage/Schedule.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TimeSlot from '../SchedulePage/Timeslot';
+
+const hothDate = '2021-02-23'; // replace with real date later
+const utcOffset = '-08:00'; // PST
+const events = [
+	{
+		startTime: new Date(hothDate + 'T09:30:00' + utcOffset),
+		events: [
+			{
+				name: 'ACM-W X ACM Hack | Hackathon 101 + Team Formation',
+				location: 'Sycamore Room',
+				description: <>First hackathon? We’ll show you all the tips and tricks you’ll
+				need to know to be successful and most importantly, have fun!</>
+			}
+		]
+	},
+	{
+		startTime: new Date(hothDate + 'T10:00:00' + utcOffset),
+		events: [
+			{
+				name: 'Git and Github',
+				location: 'De Neve Auditorium',
+				description: <>With multiple developers each working on
+				a different feature of a project, bringing things together
+				can get messy pretty easily. Come learn how to use Git to
+				manage code across multiple collaborators and versions!</>
+			},
+			{
+				name: 'INTRO TO IOS',
```

Comment:
Always use normally-cased text for accessibility. If you want the text to show up as all-caps in the UI, set the `text-transform: uppercase` CSS property instead: https://github.com/uclaacm/hoth.uclaacm.com/blob/e9119ba6af7eba66eca17d2912c8a4975f7d09a0/src/components/Footer/Footer.js#L35